server/src/main/java/org/opensearch/rest/RestHandler.java

```diff
@@ -44,6 +44,8 @@
 /**
  * Handler for REST requests
  *
+ * If new methods are added to this interface they must also be added to {@link DelegatingRestHandler}
+ *
```

Comment:
I don't think we need this comment, there will be a compile error

Reply:
It would only give a compile error if there were was not a default implementation for the method. I think it makes sense to keep the comment.